lib/elixir/test/elixir/stream_test.exs

```diff
@@ -494,9 +494,11 @@ defmodule StreamTest do
 
   test "interval/1" do
     stream = Stream.interval(10)
-    now = :os.timestamp()
-    assert Enum.take(stream, 5) == [0, 1, 2, 3, 4]
-    assert :timer.now_diff(:os.timestamp(), now) >= 50000
+
+    {time_µs, value} = :timer.tc(fn -> Enum.take(stream, 5) end)
```

Comment:
`µs` is awesome but also super yuck 😄 I am not sure we should use it in the Elixir codebase outside of testing for Unicode variables because it might make it harder to contribute or to understand why this works for newcomers. Thoughts?

Reply:
Yeah, let's go with `time_us`. :)

Reply:
Changed :)